Source/GUI/MemWatcher/MemWatchModel.cpp

```diff
@@ -666,7 +667,7 @@ MemWatchModel::CTParsingErrors MemWatchModel::importRootFromCTFile(QFile* const
 
 void MemWatchModel::writeRootToJsonRecursive(QJsonObject& json) const
 {
-  m_rootNode->writeToJson(json);
+  m_rootNode->writeToJson(json, !SConfig::getInstance().getCollapseGroupsOnSave());
```

Comment:
I don't like that I had to use SConfig here, but I couldn't think of a cleaner solution